src/api/request.ts

```diff
@@ -31,7 +31,13 @@ export interface IFetch {
   stringifyBody?: boolean;
 }
 
-const apiURL = process.env.KAOTO_API;
+let apiURL = process.env.KAOTO_API;
+console.log(process.env.NODE_ENV);
+
+if (process.env.NODE_ENV == 'production' && undefined != window.KAOTO_API) {
```

Comment:
Sorry what I meant was changing this line to this: `if (process.env.NODE_ENV == 'production' &&  window.KAOTO_API) {` because I think it would check that `window.KAOTO_API` would not be null or undefined, you don't have to explicitly check that it's not equal to `undefined`. See here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_NOT#syntax